tools/dep_updaters/update-postject.sh

```diff
@@ -20,3 +20,10 @@ NPM="$ROOT/deps/npm/bin/npm-cli.js"
 "$NODE" "$NPM" init --yes
 
 "$NODE" "$NPM" install --no-bin-links --ignore-scripts postject
+
+# TODO(RaisinTen): Replace following with $WORKSPACE
+cd ../../..
+rm -rf deps/postject
+mkdir deps/postject
```

Comment:
Any reason for not doing this?

```suggestion
rm -rf deps/postject/*
```

Reply:
@lpinca I used this script to generate the `deps/postject` directory for the first time. So although your suggestion definitely works, it wouldn't work for the case where we're generating the directory for the first time. Since generating the directory for the first time is only relevant for this PR, I don't feel strongly about either approaches, so if you feel that it needs to be done this way, let me know and I'll do it.